docs/docs/contextual-conversations.mdx

```diff
@@ -6,6 +6,18 @@ title: Complex Contextual Conversations
 
 <a aria-hidden="true" tabIndex="-1" className="anchor enhancedAnchor" id="more-complex-contextual-conversations"></a>
 
+
+
+# WHAT IS THIS PATTERN: users ask things that depend on previous convo turns to be correctly handled
```

Comment:
had to read this one a couple of times :) 

what about, 
Answering the user's request requires knowledge or context from previous conversation turns

would also keep the construction parallel with other section headings (starts with an -ing verb)

Reply:
oh ja no these headings are very much not human speak at this point, they're more just notes